tls/tls_ticket.c

```diff
@@ -351,7 +357,8 @@ ttls_tickets_configure(TlsPeerCfg *cfg, unsigned long lifetime,
 	}
 
 	timer_setup(&tcfg->timer, ttls_ticket_rotate_keys, 0);
-	secs = tcfg->lifetime - (tfw_current_timestamp() % tcfg->lifetime);
+	tfw_current_timestamp_real(&ts);
```

Comment:
These are definitely not all the places where we get timestamps, just today I commented a similar place in #2339. @const-t it seems #2339 will be merged later, so could you please fix this in your PR, but @MorganaFuture please check the whole FW, DB and TLS code for timestamps